apps/terraform/terraform.talon

```diff
@@ -0,0 +1,11 @@
+tag: terminal
+and tag: user.terraform
+-
+terraform: "terraform "
+
+terraform init: "terraform init\n"
+terraform init upgrade: "terraform init -upgrade\n"
+terraform plan: "terraform plan\n"
+terraform apply: "terraform apply "
+terraform destroy: "terraform destroy "
```

Comment:
Probably smart not to have a newline here 😅

Reply:
Yeah, that was the reasoning behind not including the newline in the modifying calls. Do you think I should remove the newline from `plan` as well?

Reply:
Unless you expect to run plan with arguments I think it's fine.

Reply:
Thanks. I thought about that, but in my experience the parameterless call is the most used one.